### PR TITLE
change gulp task for svg to create a svg json, remove a duplicate svg

### PR DESCRIPTION
**Pull request by Unetitebiere**

### CODE REVIEW

**Comment by wfortin** [comment]
If the output is in /dist it should be in the repo as we discussed today.

Other than than, it looks good.

**Comment by wfortin** [comment]
Oh i see the file now, it's just so big that github dosen't display it :stuck_out_tongue: 

https://raw.githubusercontent.com/coveo/styleguide/gulp-task-create-svg-json/dist/svg/CoveoStyleGuideSvg.json

**Comment by Unetitebiere** [comment]
Yep, I tried to remove `xmlns:xlink`with attrs and with the plugin `removeEditorsNSData` but it's alway in the output json.

**Comment by wfortin** [comment]
Remove all /target files. They are not needed are from the old version of the styleguide.

File: styleguide/index.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
```
Comment (wfortin) [inline styleguide/index.html]: This file should be removed...

**Pull request closed without merging**